setup.py

```diff
@@ -35,6 +35,7 @@
             ],
         'test': [
             'nose',
+            'openfisca-dummy-country'
```

Comment:
OK for no version constraint for `openfisca-dummy-country` if `pip` is able to find a valid version of `openfisca-dummy-country` when doing `pip install openfisca-core[test]`.